### PR TITLE
Fixing retry bundle

### LINKED ISSUE #20974: Retry bundle does not work
This is from the status popup and the listing screen

### PR DESCRIPTION
**Pull request by freddyucv**
Broke from 21.05

The retry fail because it not found the bundle folder (we are not creating the folder anymore now we are creating the tar.gzip file directly) and it was looking for the bundle.xml file to get the operation, now that value is get from the manifest file.

Also we was deleting the tar.gzip file after a retry, no matter if it was success or failed, now we are keeping the bundle's file until the BinaryCleanupJob delete it.

Also now we are not showing the retry button in the "view_publish_audit_detail.jsp" unless the bundle can b retryed

### CODE REVIEW

**Review by erickgonzalez — APPROVED** [review]

File: dotCMS/src/integration-test/java/com/dotcms/publishing/manifest/ManifestReaderFactoryTest.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+public class ManifestReaderFactoryTest {
+
+    @BeforeClass
+    public static void prepare() throws Exception {
+        IntegrationTestInitService.getInstance().init();
+    }
+
+    @Test
+    public void createCSVManifestReader() throws DotPublishingException {
```
Comment (dsilvam) [inline dotCMS/src/integration-test/java/com/dotcms/publishing/manifest/ManifestReaderFactoryTest.java:37]: missing doc

File: dotCMS/src/main/java/com/dotcms/publishing/BundlerUtil.java [hunk 2]
```diff
@@ -68,6 +91,81 @@ public static File getStaticBundleRoot(String name, boolean createDir) {
         return dir;
     }
 
+    public static boolean isRetryable(final PublishAuditStatus status) {
+        return UtilMethods.isSet(status) && isRetryable(status.getStatus());
+    }
+
+    /**
+     * Return true if a bundle ca ne retry.
+     * A bundle cab be retry if:
```
Comment (dsilvam) [inline dotCMS/src/main/java/com/dotcms/publishing/BundlerUtil.java:100]: fix javadoc misspelling

**Review by dsilvam — APPROVED** [review]

File: dotCMS/src/integration-test/java/com/dotcms/publishing/manifest/ManifestReaderFactoryTest.java (hunk 1 above)
Comment (freddyucv) [reply dotCMS/src/integration-test/java/com/dotcms/publishing/manifest/ManifestReaderFactoryTest.java:45]: done https://github.com/dotCMS/core/pull/21028/commits/e5f6254901938c4530ee9fb79c12e05db1ebb3b6#diff-01399589856f9b680a3d2b51c17ac1d7b336d979ac74e34e969b3e969f7bf14aR43

File: dotCMS/src/main/java/com/dotcms/publishing/BundlerUtil.java (hunk 2 above)
Comment (freddyucv) [reply dotCMS/src/main/java/com/dotcms/publishing/BundlerUtil.java]: done https://github.com/dotCMS/core/pull/21028/commits/e5f6254901938c4530ee9fb79c12e05db1ebb3b6#diff-827653c0f7910bf99c00088d8a5123e24ca2bf4288dff44aa8b54e067f1535b5R100

**Review by dsilvam — APPROVED** [review]

**Pull request merged**